.github/workflows/create-gh-issue-and-slack-message.yml

```diff
@@ -0,0 +1,40 @@
+name: Create-Notifications
+
+# Controls when the workflow will run
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_call:
+
+jobs:
+  Create-Notifications:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
```

Comment:
I do not think you need to checkout the code in this job

Reply:
It is needed to create the git issue (and then the slack message).

Reply:
Unless we want to hard code that in.